http/api/transport.go

```diff
@@ -111,15 +111,28 @@ func encodeResponse(_ context.Context, w http.ResponseWriter, response interface
 }
 
 func encodeError(_ context.Context, err error, w http.ResponseWriter) {
+	e, ok := status.FromError(err)
```

Comment:
Infline with if and move to the `default` case of the switch below.

Reply:
Can you explain why we should remove `default` case? If auth server fails, default is an internal server error, which makes our service unavailable.